src/platform/mock-local-home-platform.ts

```diff
@@ -106,39 +37,18 @@ export class MockLocalHomePlatform implements MockUDPListener {
     return this.localDeviceIds;
```

Comment:
Do we care about surfacing the whole mapping? or should we surface assertion helper instead ex: `isDeviceRegistered()` ?

Reply:
Done.